### DISCUSSION TITLE
Escape double quotes

### DISCUSSION THREAD

**Discussion by kiwiingenuity** [RehanSaeed/Schema.NET#10] [Q&A]
I'm experiencing difficulty deploying this library.
I'm writing a ASP.NET Core Razor Web App.
If I submit your Website example to my application it returns
```
{
	\"@context\":\"http://schema.org\",
	\"@type\":\"WebSite\",\"name\":\"Your Site Name\",
	\"alternateName\":\"An Alternative Name\"
	,\"url\":\"https://example.com\"
}
```
When I place this content onto my web page (cshtml) it comes out looking like this:

```
{
	&quot;@context&quot;:&quot;http://schema.org&quot;,
	&quot;@type&quot;:&quot;WebSite&quot;,&quot;name&quot;:&quot;Your Site Name&quot;,&quot;alternateName&quot;:
	&quot;An Alternative Name&quot;,
	&quot;url&quot;:&quot;https://example.com&quot;
}

```
What am I doing wrong?

**Comment by RehanSaeed** [comment]
You need to post some code for me to understand what you are doing.

**Comment by kiwiingenuity** [comment]
I was attempting to use this library in an ASP.NET Core Razor application and struck a problem that the .ToString() function returned escape characters.
The solution to my problem was to use @HTML.Raw().

So let say you want to place Website Schema information in your _Layout.cshtml, then you would do the following:

```
<!DOCTYPE html>
<html>
<head>
	...

    @{
        var website = new Schema.NET.WebSite()
        {
            AlternateName = "An Alternative Name",
            Name = "Your Site Name",
            Url = new Uri("https://example.com")
        };
    }

    <script type="application/ld+json">
        @Html.Raw(website.ToString())
    </script>
	
	...
</head>
...
</html>
```
